Wikipedia/Code/WMFModalImageGalleryViewController.m

```diff
@@ -478,6 +525,13 @@ - (void)updateImageForCell:(WMFImageGalleryCollectionViewCell*)cell
         [self setPlaceholderImage:download.image ofInfo:infoForImage forCellAtIndexPath:indexPath];
     }]
     .catch(^(NSError* error) {
+        @strongify(self);
+        BOOL const isInvalidURL =
```

Comment:
will double check on this, could be cases where we couldn't fetch the high-res URL (or metadata) for an image due to filename parsing issues